README.md

```diff
@@ -52,9 +52,15 @@ development:
 
 ### Create your database
```

Comment:
@hsgubert I agree regarding the verbosity of this. My thoughts were to shorten this substantially:

* Have a few short sections on the topics at hand (database configuration, migration files, migration dsl, rake tasks, query language, forked servers).
* Have at most one example per section, and link to a wiki page if more detail or examples are needed (i.e. DSL, rake tasks, and query language, as of now)

If you like this approach, I'll take a stab at it.